# irl feature parameters
irl_features:
    max_msg_age: 0.1
    target_id: 1
    
    # feature_type: "helbing"
    # feature_params: "{'radius': 3}"
    # weights: "[0.1, 0.2, 0.3, 0.4]"

    feature_type: "anisotropic"
    feature_params: "{'radius': 13}"
    weights: "[0.3, 0.7]"

    # feature_type: "icra14"
    # feature_params: "{'radius': 10}"
    # weights: "[0.9, 0.5, 0.1, 0.9, 0.5, 0.1, 0.9, 0.5, 0.1, 0.9, 0.5, 0.1]"

    cell_size: 1.0
    x1: 0.0
    y1: 0.0
    x2: 110.0
    y2: 70.0

    goal: "[68.0, 52.0, 1.0, 0.0]"
    speed: 1.0
    goal_threshold: 1.0

    # flag for the robot to start moving (enabled after wait)
    move_robot: 100.0

    # flag for teleoperation mode
    teleop_state: 0.0
